stan/math/opencl/opencl_context.hpp

```diff
@@ -107,7 +107,7 @@ class opencl_context_base {
         in_order_ = CL_TRUE;
       }
       int thread_block_size_sqrt
-          = static_cast<int>(sqrt(static_cast<double>(max_thread_block_size_)));
+          = static_cast<int>(std::sqrt(static_cast<double>(max_thread_block_size_)));
```

Comment:
[optional]
With an explicit qualification for `std::sqrt`, we shouldn't need the cast to `double`---the implementation of `std::sqrt` in C++11 has all the primitives as overloads.  See:  https://en.cppreference.com/w/cpp/numeric/math/sqrt

Reply:
Done! Thanks.